docs/edex/new-grid.md

```diff
@@ -15,105 +17,52 @@ If the parameters in the grib file haven't been previously specified, another ch
 
 ## Ingest an Unsupported Grid
 
-### Grib Products
+### Download Test Data
 
-1. Download an example grib1 file and rename to a `*.grib` extension, then copy to the manual ingest point `/awips2/data_store/ingest/` 
+Download an example grib2 file (make sure the extension is `.grib2` or EDEX may not recognize it), and then copy to the manual ingest point `/awips2/data_store/ingest/` 
 
-        wget https://downloads.unidata.ucar.edu/awips2/current/files/14102318_nmm_d01.GrbF00600 -O wrf.grib
-    
-        cp wrf.grib /awips2/data_store/ingest/
-    
-    Remember that the data distribution file (`/awips2/edex/data/utility/common_static/base/distribution/grib.xml`) will match filenames which have the `*.grib*` extension.
+    wget https://downloads.unidata.ucar.edu/awips2/current/files/CPTI_00.50_20180502-000144.grib2 -O cpti.grib2
 
-2. Confirm that the grib file decodes in the grib log file:
-    
-        edex log grib
-    
-        INFO [Ingest.GribDecode] /awips2/data_store/ingest/grib/20141026/14/wrf.grib processed in: 0.1200 (sec) Latency: 21.8080 (sec)
-        INFO [Ingest.GribDecode] /awips2/data_store/ingest/grib/20141026/14/wrf.grib processed in: 0.1180 (sec) Latency: 21.8140 (sec)
-        INFO [Ingest.GribDecode] /awips2/data_store/ingest/grib/20141026/14/wrf.grib processed in: 0.4230 (sec) Latency: 21.8360 (sec)
-        INFO [Ingest.GribDecode] /awips2/data_store/ingest/grib/20141026/14/wrf.grib processed in: 0.2240 (sec) Latency: 21.9140 (sec)
-        
-        ...
+    cp cpti.grib2 /awips2/data_store/ingest/
 
-3. Check that the hdf5 data directory exists for our unnamed grid
+Remember that the data distribution file (`/awips2/edex/data/utility/common_static/base/distribution/grib.xml`) will match filenames which have the `*.grib*` extension.
 
-        ls -latr /awips2/edex/data/hdf5/grid/GribModel:7:0:89
+### Check Grib Logs
 
-    Though the grib file has been decoded, it has been given a generic name with its center, subcenter, and process IDs (7, 0, 89, respectively). 
+Confirm that the grib file decodes in the grib log file:
     
-### Grib2 Products
-
-1. Download an example grib2 file and rename to a `*.grib2` extension, then copy to the manual ingest point `/awips2/data_store/ingest/` 
+    edex log grib
```

Comment:
I know this was in here originally, but I think the user would have to run that before they ingest the file. It might be better to tell them to just look at the log file (/awips2/edex/logs/edex-ingestGrib-YYYYMMDD.log)